sources/packages/forms/src/form-definitions/sfaa2025-26.json

```diff
@@ -10905,6 +11033,50 @@
               "lazyLoad": false,
               "tags": [],
               "hideLabel": true
+            },
+            {
+              "autofocus": false,
+              "input": true,
+              "tableView": false,
+              "inputType": "radio",
+              "label": "Is your parent(s)/step-parent/sponsor/legal guardian a resident of B.C.?",
+              "key": "parentbcResident",
+              "values": [
+                {
+                  "value": "yes",
+                  "label": "Yes",
+                  "shortcut": ""
+                },
+                {
+                  "value": "no",
+                  "label": "No",
+                  "shortcut": ""
+                }
+              ],
+              "defaultValue": "",
+              "protected": false,
+              "fieldSet": false,
+              "persistent": true,
+              "hidden": false,
+              "clearOnHide": true,
+              "validate": {
+                "required": true,
+                "custom": "",
+                "customPrivate": false,
+                "customMessage": ""
+              },
+              "type": "radio",
+              "labelPosition": "top",
+              "optionsLabelPosition": "right",
+              "tags": [],
+              "conditional": {
+                "show": "",
+                "when": null,
+                "eq": ""
+              },
+              "properties": {},
+              "lockKey": true,
+              "customConditional": "show = data.dependantstatus === \"dependant\";"
```

Comment:
Thanks for making the change. Sorry that I forgot to mention that, same thing can be achieved by simple condition, rather than javascript. 

![image](https://github.com/user-attachments/assets/193b7604-7a2e-4c14-a448-d2c7b867ca36)